src/Form/Extension/Shipping/ShippingMethodTypeExtension.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Coliship plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusColishipPlugin\Form\Extension\Shipping;
+
+use Sylius\Bundle\ShippingBundle\Form\Type\ShippingMethodType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+class ShippingMethodTypeExtension extends AbstractTypeExtension
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('colishipProductCode', TextType::class, [
+                'label' => 'monsieurbiz_coliship.form.shipping_method.coliship_product_code',
+                'help' => 'monsieurbiz_coliship.form.shipping_method.coliship_product_code_help',
+                'required' => false,
+            ])
```

Comment:
No validation at all?

Reply:
What do you propose ?
The field is not required and there is no special validation to apply on it.